frontend/src/index.pug

```diff
@@ -16,6 +16,8 @@ html
 
     script(src="https://use.fontawesome.com/releases/v5.0.13/js/all.js", defer=True)
     script(src="https://cdn.jsdelivr.net/npm/vue@2.5/dist/vue.js")
+    script(src="https://unpkg.com/vuex@3.1.3/dist/vuex.js")
+    script(src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.js")
```

Comment:
whats the reason for adding this script?

Reply:
from https://vuex.vuejs.org/installation.html
Vuex requires Promise. If your supporting browsers do not implement Promise (e.g. IE), you can use a polyfill library, such as es6-promise.

Reply:
@eugenepeh can I get your opinion on this?

Reply:
@eugenepeh can I get your opinion on this?

Reply:
I see. Can we separate the indirect dependencies, grouping them together on top?
E.g. 
```
// dependencies 
script(src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.js")

script(src="https://unpkg.com/vuex@3.1.3/dist/vuex.js")
```

Others should then be arranged lexicographically according to cdn.

Reply:
I have put the es6 dependencies to the top. I moved the font awesome package to indirect dependencies also as now we use vue-font-awesome icon, and it exploits font-awesome package. Is this ok?